core/src/main/java/hudson/cli/listeners/CliContext.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2025
```

Comment:
```suggestion
 * Copyright (c) 2025, CloudBees, Inc.
```

Should be applied to all new files.
With or without the (c), not totally sure due to existing occurrences.